### ISSUE TITLE
Milestone 4-Submission

### ISSUE THREAD

**Issue by hao44le** [ideafamily/Emonar#15]

**Comment by hatu87** [comment]
Hi @hao44le , did you split works to your teammates? Please submit your work break down to github using github issues.

**Comment by hao44le** [comment]
Yes, we split the works. And our progresses are shown in the gif. For example, API Integration is almost done. For audio I/O part, we can do single audio I/O right now. Need to figure out how to do multiple Audio File I/O for the next step. UI is like only 20% complete. We will work on UI part more as well. Thanks.
